pkg/services/accesscontrol/resourcepermissions/api.go

```diff
@@ -146,6 +172,19 @@ type setPermissionsCommand struct {
 	Permissions []accesscontrol.SetResourcePermissionCommand `json:"permissions"`
 }
 
+// swagger:route POST /access-control/:resource/:resourceID/users/:userID enterprise,access_control setResourcePermissionsForUser
+//
+// Set resource permissions for a user.
+//
+// Assigns permissions for a resource by a given type (`:resource`) and `:resourceID` to a user.
```

Comment:
nit: not sure if we want to make this implementation detail explicit, but it might be helpful.
```suggestion
// Assigns permissions for a resource by a given type (`:resource`) and `:resourceID` to a user or a service account.
```

Reply:
Done! I knew they were stored in the same place but didn't know you could send a service account ID as a user ID on these endpoints.

Reply:
Thanks! And I know, the service account / user overlapp is quite odd in places 🙈